.github/workflows/cron-e2e.yml

```diff
@@ -21,9 +23,9 @@ env:
   E2E_PASSWORD: ${{ secrets.E2E_PASSWORD }}
   E2E_ADMIN_EMAIL_IDENTITY: ${{ secrets.E2E_ADMIN_EMAIL_IDENTITY }}
   E2E_ADMIN_PASSWORD_IDENTITY: ${{ secrets.E2E_ADMIN_PASSWORD_IDENTITY }}
-  USERNAME_ADF: ${{ secrets.E2E_USERNAME }}
+  USERNAME_ADF: ${{ secrets.E2E_ADMIN_EMAIL_IDENTITY }}
```

Comment:
@maurizio-cacace why did you changed this value? 🤔

Reply:
USERNAME_ADF=$E2E_USERNAME
 and : E2E_USERNAME: ${{ secrets.E2E_ADMIN_EMAIL_IDENTITY }}
this substitution was done in the env-facade action before its removal